src/main/java/frc/team5115/Commands/Auto/VisionAuto/TurnWVision.java

```diff
@@ -0,0 +1,80 @@
+  package frc.team5115.Commands.Auto.VisionAuto;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.team5115.Classes.Accessory.Angle;
+import frc.team5115.Classes.Software.Drivetrain;
+import frc.team5115.Classes.Software.PhotonVision;
+   
+public class TurnWVision extends CommandBase{
+    private final Timer grandTimer;
+    private final TurnWVision turnWVision;
+    private Pose2d start;
+    private boolean turned = false;
+    private PhotonVision photonVision;
+    private final double angle;
+    private Drivetrain drivetrain;
+    private double photonPoseEstimator;
+    private boolean doneMoving;
+    private double dist;
+    private double speed;
+
+
+
+    public TurnWVision(TurnWVision turnWVision, PhotonVision photonVision, double absoluteAngle, double photonPoseEstimator) {
+        this.turnWVision = turnWVision;
+        angle = Angle.rollover(absoluteAngle, -180);
+        grandTimer = new Timer();
+        photonVision = new PhotonVision();
+        this.dist = dist;
+        this.speed = speed;
+        photonPoseEstimator = photonPoseEstimator;
```

Comment:
See the previous comment. Make this
`this.photonPoseEstimator = photonPoseEstimator;`